portal/server/controllers/dashboard/facilities/template-filters.js

```diff
@@ -26,20 +26,27 @@ const typeFilters = (submittedFilters) => {
  * Create filters array for the 'facility stage' field.
  * This will used in the checkboxes component 'items' array.
  */
-const hasBeenIssuedFilters = (submittedFilters) => {
-  const fieldName = FIELD_NAMES.FACILITY.HAS_BEEN_ISSUED;
+const stageFilters = (submittedFilters) => {
+  const fieldName = FIELD_NAMES.FACILITY.STAGE;
 
   const fieldInputs = [
     {
       text: BESPOKE_FILTER_VALUES.FACILITIES.ISSUED,
-      value: FACILITY_HAS_BEEN_ISSUED.ISSUED,
+      value: BESPOKE_FILTER_VALUES.FACILITIES.ISSUED,
     },
     {
       text: BESPOKE_FILTER_VALUES.FACILITIES.UNISSUED,
-      value: FACILITY_HAS_BEEN_ISSUED.UNISSUED,
+      value: BESPOKE_FILTER_VALUES.FACILITIES.UNISSUED,
     },
   ];
 
+  if (isTfmDealCancellationFeatureFlagEnabled()) {
```

Comment:
Is this new `risk_expired` the only additional status to issued/unissued?

I know `Issued` and `unissued` status are derived from the database as `hasBeenIssued` (or similar)... Is this the first new status apart from these two?

If it is -- We should really think about a migration. Or at least raise a ticket to migrate this data...Or consider our options...

Reply:
I think currently we dont need a migration, but yes I'd love to migrate all these deals from `hasBeenIssued` to just using `facilityStage`. This could be a bit trickier on bss/ewcs deals since they already have `facilityStage` and it takes some other values that I dont understand.
Happy to chat this through if you're concerned about this but (hopefully) I've not done anything needing a migration (yet!)